internal/exec/help.go

```diff
@@ -20,43 +19,59 @@ func processHelp(
 		u.PrintMessage(fmt.Sprintf("atmos %s <command> <component> -s <stack> [options]", componentType))
 		u.PrintMessage(fmt.Sprintf("atmos %s <command> <component> --stack <stack> [options]", componentType))
 
-		if componentType == "terraform" {
-			u.PrintMessage("\nAdditions and differences from native terraform:")
-			u.PrintMessage(" - before executing other 'terraform' commands, 'atmos' runs 'terraform init'")
-			u.PrintMessage(" - you can skip over atmos calling 'terraform init' if you know your project is already in a good working state by using " +
-				"the '--skip-init' flag like so 'atmos terraform <command> <component> -s <stack> --skip-init")
-			u.PrintMessage(" - 'atmos terraform deploy' command executes 'terraform apply -auto-approve' (sets the '-auto-approve' flag when running 'terraform apply')")
-			u.PrintMessage(" - 'atmos terraform deploy' command supports '--deploy-run-init=true/false' flag to enable/disable running 'terraform init' " +
-				"before executing the command")
-			u.PrintMessage(" - 'atmos terraform apply' and 'atmos terraform deploy' commands support '--from-plan' flag. If the flag is specified, " +
-				"the commands will use the planfile previously generated by 'atmos terraform plan' command instead of generating a new planfile")
-			u.PrintMessage(" - 'atmos terraform apply' and 'atmos terraform deploy' commands commands support '--planfile' flag to specify the path " +
-				"to a planfile. The '--planfile' flag should be used instead of the planfile argument in the native 'terraform apply <planfile>' command")
-			u.PrintMessage(" - 'atmos terraform clean' command deletes the '.terraform' folder, '.terraform.lock.hcl' lock file, " +
-				"and the previously generated 'planfile', 'varfile', and 'backend.tf.json' file for the specified component and stack. " +
-				"Use the --everything flag to also delete the Terraform state files and directories for the component. " +
-				"Note: State files store the local state of your infrastructure and should be handled with care, if not using a remote backend.\n\n" +
-				"Additional flags:\n" +
-				"  --force Forcefully delete Terraform state files and directories without interaction\n" +
-				"  --skip-lock-file Skip deleting the '.terraform.lock.hcl' file\n\n" +
```

Comment:
We need to make sure these are documented

Reply:
Done